src/lib/config.js

```diff
@@ -2,7 +2,9 @@ require("dotenv").config();
 
 module.exports = {
   API: {
-    BASE_URL: process.env.API_BASE_URL || "http://localhost:8090",
+    BASE_URL:
+      process.env.API_BASE_URL ||
+      "https://api-f2f-cri-api.review-o.dev.account.gov.uk",
```

Comment:
Do we wanna default this back to localhost instead of dev env @Caitlin-cooling ?